planet/cli/orders.py

```diff
@@ -77,16 +77,20 @@ async def list(ctx, state, limit, pretty):
 @translate_exceptions
 @coro
 @click.argument('order_id', type=click.UUID)
+@click.option('--ids-only', is_flag=True, help='Returns only the item IDs.')
```

Comment:
@kevinlacaille #645 is only about getting a list of ids from search. Let's not extend to the orders CLI here. Can you remove the changes to planet/cli/orders.py?